test/libevmasm/Assembler.cpp

```diff
@@ -254,6 +255,30 @@ BOOST_AUTO_TEST_CASE(immutable)
 	);
 }
 
+BOOST_AUTO_TEST_CASE(subobject_encode_decode)
+{
+	Assembly assembly;
+	auto root_asm = make_shared<CharStream>("lorem ipsum", "root.asm");
+
+	Assembly subAsm;
+	auto sub_asm = make_shared<CharStream>("lorem ipsum", "sub.asm");
+	shared_ptr<Assembly> _subAsmPtr = make_shared<Assembly>(subAsm);
+
+	Assembly subSubAsm;
+	auto sub_sub_asm = make_shared<CharStream>("lorem ipsum", "sub_sub.asm");
+	shared_ptr<Assembly> _subSubAsmPtr = make_shared<Assembly>(subSubAsm);
```

Comment:
`root_asm`, `sub_asm` and `sub_sub_asm` seem to be unused here.

Reply:
Removed